watched_nses.yml

```diff
@@ -516,7 +513,9 @@ items:
   ns: mihanwebhost.com.
 - ns: offshoreracks.com.
 - ns: sathyats.net.
-- ns: shared-host.org.
+- comment: no longert responding 2020-04-02
```

Comment:
You've substituted typos in NSes to typos in comments :)

Reply:
Thanks for noticing, I'll fix and merge on Monday.